ffplayout-engine/src/utils/mod.rs

```diff
@@ -239,6 +239,7 @@ pub fn prepare_output_cmd(
 /// map media struct to json object
 pub fn get_media_map(media: Media) -> Value {
     json!({
+        "title": media.title,
```

Comment:
Maybe here is also a way to make title optional.

Reply:
I have finished the corrections, sorry for the delay. If there are more corrections, please mention them.